components/ActionModals/ActionModal.vue

```diff
@@ -37,6 +37,7 @@
           v-model="feeDenom"
```

Comment:
this is already an implied `@change` handler, isn't it?

Reply:
maybe. You mean remove the method in ActionModal?

Reply:
OK, it works. Thanks for the hint